modules/pam_mkhomedir/mkhomedir_helper.8.xml

```diff
@@ -48,6 +51,15 @@
       <replaceable>umask</replaceable>.
     </para>
 
+    <para>
+      <replaceable>dir_mode</replaceable> defines the mode option for all
+      home directory's subfolders. Put "<emphasis>copymode</emphasis>" to keep
+      the same access permission as <replaceable>path-to-skel</replaceable>'s
+      subfolders, otherwise it will keep the default value 0777. All directories
+      access permissions are computed from the value of
+      <replaceable>umask</replaceable>.
```

Comment:
To be honest, this description looks confusing to me.  It's not clear whether this parameter can be set yo anything besides `copymode`, and how it interacts with `umask`.

Reply:
Agree. I try to simplify this one with the last modifications